module/snv_annotations.nf

```diff
@@ -117,7 +149,7 @@ process annotate_trinucleotide_BCFtools {
     container params.docker_image_bcftools
 
     publishDir path: "${params.output_dir_base}/intermediate/${task.process.replace(':', '/')}",
-        pattern: "output.vcf.gz{,.tbi}",
+        pattern: "output.vcf.gz",
```

Comment:
Does the `{,.tbi}` syntax work for the `saveAs` field? The index file wasn't being emitted properly with `saveAs: { "Trinucleotide-annotated-${sample_id}.vcf.gz" }`.

Reply:
Ohhh of course - I suspect that as-is the `Trinucleotide-annotated-${sample_id}.vcf.gz` has a 50/50 chance of being the index file instead of the VCF. My bad for writing it that way!

Ideally we should save all outputs as intermediate files, but that'll take more work that doesn't need to be part of this PR.